lib/winston-logstash-udp.js

```diff
@@ -70,11 +71,14 @@ LogstashUDP.prototype.log = function(level, msg, meta, callback) {
         json: true
     });
 
-    self.sendLog(logEntry, function() {
-        self.emit('logged', true);
-        callback(null, true);
-    });
+    // There is no need to wait for a callback as there is no guarantee that
+    // message will ever reach destination and we never get the feedback with UDP.
+    // Having self.emit('logged', true) as part of callback in sendLog would
+    // cause it to take 10% more time.
+    self.sendLog(logEntry);
```

Comment:
The callback determines if the UDP message has been sent (not whether it is received).

From the nodejs docs for `dgram.Socket.send()`:

> An optional callback may be specified to detect DNS errors or for determining when it's safe to reuse the buf object. Note that DNS lookups delay the time to send for at least one tick. The only way to know for sure that the datagram has been sent is by using a callback.

I think it is worth knowing if the system has sent the UDP message.  If it doesn't, then we know that the message will definitely not be received.

However, perhaps the callback should be optional for cases where the sent status of the UDP message is not needed/desired.